client/gogeta_client.go

```diff
@@ -13,26 +17,40 @@ import (
 	"github.com/aws/aws-sdk-go/service/sqs"
 )
 
-// GogetaClient is the high level implementation of the gogeta
-// source control system
-type GogetaClient struct {
-	Queue   queuesystem.Messages
-	Log     logger.Log
-	SCM     sourcesystem.SourceSystem
-	Storage storehouse.StoreHouse
+// Gogeta is the source control manager implementation
+type Gogeta struct {
```

Comment:
Do all of these properties need to be exposed? If they're just for testing then we could just use internal tests on it?